internal/provider/data_source_cloudexport_item.go

```diff
@@ -19,11 +18,11 @@ func dataSourceCloudExportItem() *schema.Resource {
 }
 
 func dataSourceCloudExportItemRead(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
-	tflog.Debug(ctx, fmt.Sprintf("Kentik API request - read item with ID: %s\n", d.Get("id").(string)))
+	tflog.Debug(ctx, "Get cloud export Kentik API request", "ID", d.Get("id").(string))
 	getResp, httpResp, err := m.(*kentikapi.Client).CloudExportAdminServiceAPI.
 		ExportGet(ctx, d.Get("id").(string)).
 		Execute()
-	tflog.Debug(ctx, fmt.Sprintf("Kentik API response - read item:\n%s\n", httpResp.Body))
+	tflog.Debug(ctx, "Get cloud export Kentik API response", getResp)
```

Comment:
```suggestion
	tflog.Debug(ctx, "Get cloud export Kentik API response", "response", getResp)
```
Could you show if the output of the log is correct? I think the Debug() API requires key-value  pairs after the message.

In other places too.

Reply:
Done